package.json

```diff
@@ -76,5 +77,8 @@
     "lint": "standard",
     "test": "echo 'not yet implemented, wanna help?'"
   },
+  "standard": {
+    "parser": "babel-eslint"
```

Comment:
True, for some reason, `standard` does not complain about the object-spread-operator (which is also experimental) but it does on the class-properties (stage-2). :S